web/server/vue-cli/src/assets/userguide/userguide.md

```diff
@@ -334,133 +427,356 @@ stored.
+  want to compare against the baseline filter set.
+  - `Outstanding reports on a given date` - Here you can select date which
+  you want to compare against the baseline filter set.
+  - `Diff type` - Here you can set if you'd like to see the reports which
+  appear only in the **Baseline**, **Compare to** or both.
+- `File path`: You can choose a set of files to restrict the list of bug
+reports.
+- `Checker name` - If you are interested in specific type of bugs then here
+you can choose them.
+- [`Severity`](#severity-levels) - The nature of the reports is sorted
+in different severity levels. For example, a division by zero or a null pointer
+dereference is more serious than an unused variable. By this field you can
+select the reports on the given severity levels.
+- [`Latest Review Status`](#review-status) - You can select the reports
+with the given review status to check only *False positive*, *Unreviewed*, etc.
+reports.
+- [`Latest Detection Status`](#detection-status) - You can select the
+reports with the given detection status to check only *Unresolved*, *Resolved*,
+etc. reports.
+- `Analyzer name` - If you are interested in specific type of reports which
+were detected by an analyzer (Clang Tidy, CppCheck etc) then here you can
+choose them.
+- [`Source component`](https://github.com/Ericsson/codechecker/blob/master/docs/web/user_guide.md#source-components) -
+Here you can select multiple source components which are named collection of
+directories specified as directory filter.
+- `Checker message` - The static analysis tools provide a message to indicate
+the reason of a specific bug. This message is also filterable.
+- **Dates**
+  - `Detection date` - Filter reports that were detected between the given
+  interval.
+  - `Fix date` - Filter reports that were fixed between the given interval.
+- `Report hash` - Every report has a unique (hash) identifier called Report
+Identifier (RI) which can be filtered by using this input box.
+- `Bug path length` - A bug path length interval can restrict the list of
+displayed bug reports. In these fields you can choose the minimum and maximum
+values of bug path length.
+
+Every filter is collapsed by default to save more space in the UI. To expand
+the filter and see the selected filter items you can click on the down arrow
+(`v` icon) beside each filter name.
+
+### Date filters
+Every report has a detection date and a fix date.
+- `Detection date`: the storage date when the report was stored to the server
+for the first time.
+- `Fix date`: storage date when the report is dissapeared from a run update.
+
+Available date filters:
+- `Detection date`: get results which were detected between the given dates.
+- `Fix date`: get results which were fixed between the given dates.
+- `Outstanding reports on a given date`: get results which were
+<i>DETECTED BEFORE</i> the given date and <i>NOT FIXED BEFORE</i> the given
+date.
+
+### Regex based filters
```

Comment:
I don't know what's the official name of the `*` placeholder, but it is not a regex. Regex is something more complex and we don't support that. I'll try to find its official name and use that. Also `*` is not a quantifier (in regex it is, because there it means quantity of the preceding expression) here it is just a joker character (see the text a few lines below).